lib/matplotlib/axis.py

```diff
@@ -870,6 +870,28 @@ def cla(self):
         """Clear this axis."""
         return self.clear()
 
+    def minorticks_on(self):
+        """
+        Display minor ticks on the Axis.
```

Comment:
Lets make this more explicit so users have some idea what is going on under the hood if they need to: 

```suggestion
        Display default minor ticks on the Axis, depending on the scale (`~.Axis.get_scale`). 
        
        Most scales default to `~.AutoMinorLocator`, except 'log' uses `~.LogLocator` and 
        'symlog' uses `~.SymmetricalLogLocator`.  
        
```